app/src/models/init-models.ts

```diff
@@ -475,6 +485,11 @@ export function initModels(sequelize: Sequelize) {
     foreignKey: "invitingUserId",
     as: "invitingUser",
   });
+  User.hasMany(OrganizationInvite, {
+    foreignKey: "userId",
+    as: "organizationInvites",
+  });
+  OrganizationInvite.belongsTo(User, { as: "user", foreignKey: "userId" });
```

Comment:
`Organization.hasMany(OrganizationInvite, ...)` seems to be missing
And potentially having a direct way to access the many to many relationship from Organization to User, so you can get all users that have access to an organization.

Reply:
Will add the FK to Organization.
We decided to limit a user to only one organization, and if need be they can create multiple accounts. Therefore it's a one to many and this code should be enough.